CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ActivityExtensions.cs

```diff
@@ -142,6 +147,31 @@ public static class ActivityExtensions
             return activity;
         }
 
+        /// <summary>
+        /// Notifies the mentioned users.
+        /// </summary>
+        /// <param name="replyActivity">The reply activity.</param>
+        /// <returns>Modified activity.</returns>
+        public static Activity NotifyMentionedUsers(this Activity replyActivity)
```

Comment:
Why is this limited to messages with mentioned users only? It works for 1:1 message w/ no mentioned users (and for channel messages w/ mentions isn't it automatically added to the feed for the mentioned users?)
Perhaps we should just have a method to add Notification w/ Alert=true to channel data for any IMessageActivity.